packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.types.ts

```diff
@@ -26,17 +26,17 @@ export interface ICommandBarProps extends React.HTMLAttributes<HTMLDivElement> {
   componentRef?: IRefObject<ICommandBar>;
 
   /**
-   * Items to render
+   * Items to render. ICommandBarItemProps extend IContextualMenuItem
    */
   items: ICommandBarItemProps[];
 
   /**
-   * Items to render on the right side (or left, in RTL).
+   * Items to render on the right side (or left, in RTL). ICommandBarItemProps extend IContextualMenuItem
    */
   farItems?: ICommandBarItemProps[];
 
   /**
-   * Default items to have in the overflow menu
+   * Default items to have in the overflow menu. ICommandBarItemProps extend IContextualMenuItem

```

Comment:
We have this problem in multiple components. I thought a better longer term fix may be to have the parser automatically document any interface that extends another, but I don't know how difficult that would be to do.

Reply:
Yeah. And considering the ease of doing this manually...

Reply:
Well, it becomes another maintenance issue, and they can get out of sync. For long-term I think it's best to have the parser handle this. (Or at least look at it long enough to decide it can't be done easily.)

Reply:
Is this solution okay for the interim then?

Reply:
Yeah, but I think it'd be good to create an issue for this (if there isn't one already)